tests/filters/test_blur.py

```diff
@@ -9,42 +9,58 @@
 # Copyright (c) 2011 globo.com thumbor@googlegroups.com
 
 from preggy import expect
+from tornado.testing import gen_test
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'tornado.testing'](https://app.codacy.com/manual/heynemann/thumbor/pullRequest?prid=4967909)